src/libstd/macros.rs

```diff
@@ -210,8 +210,9 @@ macro_rules! eprintln {
     })
 }
 
-/// A macro for quick and dirty debugging with which you can inspect
-/// the value of a given expression. An example:
+/// Prints the value of a given expression for quick and dirty debugging.
```

Comment:
```suggestion
/// Prints and returns back the value of a given expression for quick and dirty debugging.
```

Your change gives the impression that the macro just prints and is typed at `()` but the macro returns back the value passed. I think an essential point has been lost here.

Reply:
Good point, done.